Makefile

```diff
@@ -197,7 +197,7 @@ controller-gen: ## Download controller-gen locally if necessary.
 
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
+	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.3)
```

Comment:
I'd recommend  putting this in a different commit and adding information about all the changes in the PR message.
commit 1: change go-get-tool to use go install
commit 2: update kustomize version
commit 3: remove ginkgo

If the go.mod changes are unrelated, please send a separate PR for them

Reply:
done